core/lexicon/en/menu.inc.php

```diff
@@ -25,7 +25,7 @@
 $_lang['menu_parent'] = 'Parent Menu';
 $_lang['menu_parent_err_ns'] = 'Parent menu not specified!';
 $_lang['menu_parent_err_nf'] = 'Parent menu not found!';
-$_lang['menu_remove'] = 'Delete Menu Item';
+$_lang['menu_remove'] = 'Delete Menu';
 $_lang['menu_top'] = 'Top Menu';
```

Comment:
Maybe it makes sense to change "Top Menu" to "Main Menu"? Since in the MODX3 menu is no longer on top.